.github/workflows/main.yml

```diff
@@ -203,12 +203,6 @@ jobs:
       - name: git checkout
         uses: actions/checkout@v4
 
-      - name: setup go ${{ env.GO_VERSION }}
```

Comment:
We need `go` for this step: https://github.com/lightningnetwork/lnd/blob/643e415448fff822e8aea6d3d7e14fe467d4a4e3/scripts/check-sample-lnd-conf.sh#L21

Did you mean to remove it from the `sqlc` check? I think there we don't require it.

Reply:
yeah I got it wrong...now removed😅